tests/suite/cli-ui/rustup/rustup_help_cmd_stdout.toml

```diff
@@ -16,7 +16,7 @@ Commands:
   toolchain    Modify or query the installed toolchains
   target       Modify a toolchain's supported targets
   component    Modify a toolchain's installed components
-  override     Modify directory toolchain overrides
+  override     Modify toolchain overrides for directories
```

Comment:
These changes should be squashed into the commit that originated the code changes, so tests should pass after each commit.